tools/vargo/src/main.rs

```diff
@@ -679,6 +683,111 @@ fn run() -> Result<(), String> {
                         }
                         Ok(())
                     }
+                    Task::Fmt => {
+                        if !status.success() {
+                            return Err(format!(
+                                "`cargo fmt` returned status code {:?}",
+                                status.code()
+                            ));
+                        }
+                        if exclude.contains(&"vstd".to_owned()) {
+                            return Ok(());
+                        }
+
+                        let verusfmt_check = match std::process::Command::new("verusfmt")
+                            .arg("--version")
+                            .output()
+                        {
+                            Ok(output) => {
+                                if !output.status.success() {
+                                    return Err(format!(
+                                        "`verusfmt` returned status code {:?}",
+                                        status.code()
+                                    ));
+                                }
+                                let verusfmt_version_stdout = String::from_utf8(output.stdout)
+                                    .map_err(|_| format!("invalid output from verusfmt"))?;
+                                let verusfmt_version_re =
+                                    Regex::new(r"^verusfmt ([0-9]+)\.([0-9]+)\.([0-9]+)\n$")
+                                        .unwrap();
+                                let verusfmt_version = verusfmt_version_re
+                                    .captures(&verusfmt_version_stdout)
+                                    .ok_or(format!("invalid output from verusfmt"))?
+                                    .extract::<3>()
+                                    .1
+                                    .iter()
+                                    .map(|v| v.parse::<u64>().unwrap())
+                                    .collect::<Vec<u64>>();
+                                for (cv, ev) in
+                                    verusfmt_version.iter().zip(MINIMUM_VERUSFMT_VERSION.iter())
+                                {
+                                    if ev > cv {
+                                        return Err(format!("expected `verusfmt` version to be at least {}.{}.{}, found {}.{}.{}; refer to CONTRIBUTING.md for installation instructions",
```

Comment:
`CONTRIBUTING.md` points to GitHub for installation instructions anyways, we can just point folks there directly instead?

```suggestion
                                        return Err(format!("expected `verusfmt` version to be at least {}.{}.{}, found {}.{}.{}; refer to https://github.com/verus-lang/verusfmt for installation instructions",
```

Reply:
This was to let the reader skip a step. `CONTRIBUTING.md` points to `verusfmt` for installation instruction.

Reply:
Fwiw, I was suggesting that we change it to skip the step; I see you've already changed that manually, so marking this as resolved.